### PR TITLE
windows 10 is a bit different and we need these

### PR DESCRIPTION
**Pull request by kwccoin**
Need to
- install npm
- install curl
- install more packages
- curl need to use double quote in the curl parameters
- curl need to escape double quote in json string

### CODE REVIEW

File: win10-difference.cmd [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+rem install curl exe
+rem install npm (node.js)
+
+rem update npm 
+
+npm i npm
+
+rem install all missing modules
+
+npm install express
```
Comment (conradoqg) [inline win10-difference.cmd:19]: Instead of installing one module by one, why don't we do an npm install? It should work if it doesn't work something bad is happening.

File: win10-difference.cmd [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+npm install yargs
+
+npm install cli-color
+
+rem starting only one node (need to cancel if start > 1 node)
+
+node bin/naivecoin.js
+
+rem 2018-03-11T14:13:56.779Z - info - 1: Starting node 1
```
Comment (conradoqg) [inline win10-difference.cmd:49]: Why are these lines here? (lines showing the log of naivecoin)

**Review by conradoqg — CHANGES_REQUESTED** [review]
Could you instead of adding this file to the repo, improve the readme.md with those double quotes and `\`. Let's try to find a common ground for Linux and Windows, I use windows so I can test it here.

File: win10-difference.cmd (hunk 2 above)
Comment (kwccoin) [reply win10-difference.cmd:49]: I assume this is for learning and one thing I find it is very hard to learn if I do not have a log showing the exact response.  But may be it can be in a separate file e.g. windows-experience-log.txt

### need something to start the node but not want all may be

For those install one by one, I think I miss your step in development 
    npm install
I just jump right in forget you have done the package part

But then one must note that if one is not in development mode, install 300MB+ node package is not kind for non-developer.  But without some of these, the first node command would not work. 

### readme change seems ok for all platform 

I would try to do the readme change but that may be after I try the naivecoin on Mac as well to complete the whole exercise.  

### now my problem is that docker ubuntu nodejs does not work ... would real ubuntu work?

Actually my problem is not able to run it on docker.  Sort out the docker image running but still got a node js message

Comment (kwccoin) [reply win10-difference.cmd:49]: # docker work using your package is ok

just need one command

    docker-compose up 

work for docker image with 3 nodes and even better it is 0.0.0.0 and hence no need to change the localhost to my ip address.  One thing is that the node is all run under the same 3001 and hence only this the wallet is maintained.  
    
and that image work with mac node as well but if the node is different, the wallet cannot be re-used.

### problem with the ubuntu (docker image) where the nodejs does not like the javacript; got error

The only left is ubuntu and hence

    docker run -v ~/Downloads/naivecoin/:/naivecoin -ti ubuntu /bin/sh

I test and the command with " to  ' and escape " work for curl.  However, under docker ubuntu mode, the following error message:

  > docker run -v ~/Downloads/naivecoin/:/naivecoin -ti ubuntu /bin/sh
  > cd naivecoin
  > apt-get update
  > apt-get install nodejs
  > apt-get install npm
  > npm install
  > npm test 
  > nodejs bin/naivecoin.js  -a 192.168.1.8 -p 3003 --name 3 --peers http://192.168.1.8:3001

error message:

''''
    /naivecoin/lib/naivecoin.js:19

    let blockchain = new Blockchain(name);
    ^^^

SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:374:25)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/naivecoin/bin/naivecoin.js:2:19)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)

# 
''''

Comment (conradoqg) [reply win10-difference.cmd:49]: A lot to respond here, hehe. Please give me some time to respond. Talk to you soon.

**Comment by conradoqg** [comment]
I did a lot of changes recently. Could you check the branch v1.0.0 and see if you still have windows 10 problems? I'm using Windows 10 as well.
